lacework/resource_lacework_vulnerability_exception_host.go

```diff
@@ -0,0 +1,402 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkVulnerabilityExceptionHost() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkVulnerabilityExceptionHostCreate,
+		Read:   resourceLaceworkVulnerabilityExceptionHostRead,
+		Update: resourceLaceworkVulnerabilityExceptionHostUpdate,
+		Delete: resourceLaceworkVulnerabilityExceptionHostDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkVulnerabilityException,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the vulnerability exception",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The description of the vulnerability exception",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "The state of the vulnerability exception",
+			},
+			"reason": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "List of severities for the vulnerability exception. Valid severities are:" +
+					" False Positive, Accepted Risk, Compensating Controls, Fix Pending, Other",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(strings.Title(strings.ToLower(val.(string))))
+					},
+					ValidateFunc: func(value interface{}, key string) ([]string, []error) {
+						switch strings.ToLower(value.(string)) {
+						case "false positive", "accepted risk", "compensating controls", "fix pending", "other":
+							return nil, nil
+						default:
+							return nil, []error{
+								fmt.Errorf(
+									"%s: can only be 'False Positive', 'Accepted Risk', 'Compensating Controls', 'Fix Pending', 'Other'", key,
+								),
+							}
+						}
+					},
+				},
+			},
+			"vulnerability_criteria": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Required:    true,
+				Description: "The vulnerability criteria",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cves": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+								StateFunc: func(val interface{}) string {
+									return strings.TrimSpace(val.(string))
```

Comment:
Api doesn't have any enforcement on this. I don't think we should either to avoid a weird state. In the api you can pass the same cve with 2 different cases and they return as 2 separate items.